source/guides/dashboard/analytics.md

```diff
@@ -123,3 +116,93 @@ Unique tests and number of spec files are computed respective of the filters sel
 {% imgTag /img/dashboard/analytics/dashboard-analytics-test-suite-size-table.png "Dashboard Analytics Test Suite Size Table Screenshot" %}
 
 A table of results grouped by date for the time range filter is displayed with unique tests and spec files.
+
+## Top Failures
+
+{% imgTag /img/dashboard/analytics/dashboard-analytics-top-failures.png "Dashboard Analytics Top Failures Screenshot" %}
+
+This report shows the top failures in your test suite.
+
+### Filters
```

Comment:
We should call out the custom grouping functionality here too. Users can roll up failure rates (and test performance) by test case, spec, tag, or branch.

Reply:
Good catch. I'll update the rest of the docs